bundler/lib/bundler.rb

```diff
@@ -344,25 +344,7 @@ def clean_env
 
     # @return [Hash] Environment with all bundler-related variables removed
     def unbundled_env
-      env = original_env
-
-      if env.key?("BUNDLER_ORIG_MANPATH")
-        env["MANPATH"] = env["BUNDLER_ORIG_MANPATH"]
```

Comment:
`BUNDLER_ORIG_` appears to have been an old prefix that isn't used anymore https://github.com/rubygems/rubygems/commit/8e3b925413e1ed5f721e2f7d90dfe6f0e3c43af5#diff-83eb5e43a306cb4526be4c5e5a790479L9